pkg/v2/service.go

```diff
@@ -201,7 +202,7 @@ func (s *service) Cleanup(ctx context.Context) (*taskAPI.DeleteResponse, error)
 	if err != nil {
 		return nil, err
 	}
-	r := proc.NewRunsc(proc.RunscRoot, path, ns, runtime, nil)
```

Comment:
This looks like we want to fall back to proc.RunscRoot if a custom config is not provided. I don't see that we are doing that though.

Reply:
The runtime instantiation code (https://github.com/google/gvisor-containerd-shim/blob/b7a9207c309f6ff5df357aa5ac7b43e070ff5d80/pkg/v2/service.go#L809) already does it like that and works fine even when no configuration is specified.

Reply:
Ok. I see. It does get set to the default value if options.Root is empty.

https://github.com/google/gvisor-containerd-shim/blob/b7a9207c309f6ff5df357aa5ac7b43e070ff5d80/pkg/v1/proc/init.go#L83